python/huggingfaceserver/huggingfaceserver/model.py

```diff
@@ -88,44 +89,58 @@ def infer_task_from_model_architecture(model_config_path: str):
         if task is None:
             raise ValueError(f"Task couldn't be inferred from {architecture}. Please manually set `task` option.")
 
+    @staticmethod
+    def infer_vllm_supported_from_model_architecture(model_config_path: str):
```

Comment:
This is not used?

Reply:
Used in the init method - https://github.com/kserve/kserve/pull/3415/files#diff-d02c376ad82bad8a11ceaecb4ea815822123e34b961f29fd5050225af82e62b4R76

Reply:
Sorry I missed it